scala/private/common.bzl

```diff
@@ -26,15 +26,12 @@ def _collect_jars_when_dependency_analyzer_is_off(dep_targets):
   runtime_jars = []
 
   for dep_target in dep_targets:
+    # we require a JavaInfo for dependencies
+    # must use java_import or scala_import if you have raw files
```

Comment:
Why keep the “if”? I think you can require the divider on the signature of the attribute and so throw all of the conditionals away

Reply:
because scrooge puts everything in the deps sadly... we want to fix it, but one step at a time.

Reply:
scrooge macro puts all the thrift dependencies on the deps. :( we need #510